content/08-cardano-sidechains/03-example-evm-sidechain/04-sidechain-token.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: SC_Token and faucet
+metaTitle: SC_Token and faucet
+---
+
+## What is SC_Token?
+
+**SC_Token<sup>1</sup>** is the native token of Cardano sidechains launched through a Cardano EVM sidechain client. SC_Token can be moved between the main chain (Cardano) and the example EVM sidechain, and SC_Token is required to pay for gas.
```

Comment:
@stevanlohja @NeilBurgess42 not sure in 

> **the native token of Cardano sidechains** launched through a Cardano EVM sidechain client.

this is a correct phrasing. This can be interpret as the SC_Token is the token for ALL sidechains. 

Perhaps just say 
> the native token of Cardano EVM sidechain testnet.